src/main/java/dk/sdu/seb05/semesterprojekt/DomainLayer/Session.java

```diff
@@ -0,0 +1,31 @@
+package dk.sdu.seb05.semesterprojekt.DomainLayer;
+
+
+public class Session {
+    public int producerID;
+    public boolean isAdmin;
+
+    public Session(){
+        this.producerID = 0;
+        this.isAdmin = false;
+    }
+
+    public Session(int producerID){
+        this.producerID = producerID;
+        this.isAdmin = false;
+    }
+
+    public Session(boolean isAdmin){
+        this.producerID = 0;
```

Comment:
Jeg tænker det nok er bedst at producerId er -1 for en admin. For det er ikke muligt at få det producerId på andre måder. 
```suggestion
        this.producerID = -1;
```